openapi/auth-server.yaml

```diff
@@ -362,51 +500,43 @@ components:
-          description: 'A unique access token for continuing the request, called the "continuation access token".'
-          required:
-            - value
-          properties:
-            value:
-              type: string
-        uri:
-          type: string
-          format: uri
-          description: The URI at which the client instance can make continuation requests.
-        wait:
-          type: integer
-          description: The amount of time in integer seconds the client instance MUST wait after receiving this request continuation response and calling the continuation URI.
-      required:
-        - access_token
-        - uri
-    access_token:
-      title: access_token
-      type: object
-      description: A single access token or set of access tokens that the client instance can use to call the RS on behalf of the RO.
+    interval:
+      title: Interval
+      type: string
+      description: '[ISO8601 repeating interval](https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals)'
+      examples:
+        - 'R11/2022-08-24T14:15:22Z/P1M'
+        - 'R/2017-03-01T13:00:00Z/2018-05-11T15:30:00Z'
+        - 'R-1/P1Y2M10DT2H30M/2022-05-11T15:30:00Z'
+    limits-outgoing:
+      title: limits-outgoing
+      description: Open Payments specific property that defines the limits under which outgoing payments can be created.
+      type: [object, 'null']
       properties:
-        value:
-          type: string
-          description: The value of the access token as a string.  The value is opaque to the client instance.  The value SHOULD be limited to ASCII characters to facilitate transmission over HTTP headers within other protocols without requiring additional encoding.
-        manage:
-          type: string
-          format: uri
-          description: The management URI for this access token. This URI MUST NOT include the access token value and SHOULD be different for each access token issued in a request.
-        expires_in:
-          type: integer
-          description: The number of seconds in which the access will expire.  The client instance MUST NOT use the access token past this time.  An RS MUST NOT accept an access token past this time.
-        access:
-          $ref: ./schemas.yaml#/components/schemas/access
-      required:
-        - value
-        - manage
-        - access
-      additionalProperties: false
+        receiver:
+          $ref: './schemas.yaml#/components/schemas/receiver'
+        sendAmount:
+          $ref: './schemas.yaml#/components/schemas/amount'
+        receiveAmount:
+          $ref: './schemas.yaml#/components/schemas/amount'
+        interval:
+          $ref: '#/components/schemas/interval'
+      anyOf:
+        - not:
+            required:
+              - interval
+        - required:
+            - sendAmount
+        - required:
+            - receiveAmount
+    list-actions:
+      enum:
+        - list
+        - list-all
+    read-actions:
```

Comment:
Do we need `list-actions` & `read-actions` if they aren't used?

Reply:
> Do we need `list-actions` & `read-actions` if they aren't used?

What do you mean?

Reply:
L523 `list-action` & L536 `read-actions` schemas aren't referenced anywhere in the repo, hence the GH Actions warning

Reply:
Oh, I didn't realize that. I'll investigate